### PR TITLE
Remove ModMenu stuff

### PR DESCRIPTION
**Pull request by DragonEggBedrockBreaking**
Both `modmenu:clientsideOnly` and `modmenu:api` are deprecated and unneeded, and also warn in logs that they will be removed in 1.18 snapshots and that we should tell the mod authors to support the new API.

### CODE REVIEW

**Comment by DragonEggBedrockBreaking** [comment]
fix: i had forgotten to add the 'library' tag to the api

File: src/api/resources/fabric.mod.json [hunk 1]
```diff
@@ -25,9 +25,10 @@
   "depends": {
     "fabricloader": ">=0.4.0"
   },
-
+
```
Comment (Earthcomputer) [inline src/api/resources/fabric.mod.json:28]: Can you revert the change to this line?

Comment (DragonEggBedrockBreaking) [reply src/api/resources/fabric.mod.json]: done

**Review by Earthcomputer — APPROVED** [review]
Thanks! It likely won't make it into another 1.16.5 release unless there is a critical issue that needs fixing.

**Comment by DragonEggBedrockBreaking** [comment]
> Thanks! It likely won't make it into another 1.16.5 release unless there is a critical issue that needs fixing.

As long as there is a release before 1.18, it should be fine.
